EIPS/eip-6786.md

```diff
@@ -0,0 +1,105 @@
+
+## Motivation
+
+There are many marketplaces which do not enforce any royalty payment to the NFT creator every time the NFT is sold or re-sold and/or providing a way for doing it. There are some marketplaces which use specific system of royalties, however that system is applicable for the NFTs creates on their platform.
+
+In this context, there is a need of a way for paying royalties, as it is a strong incentive for creators to keep contributing to the NFTs ecosystem.
+
+Additionally, this standard will provide a way of computing the amount of royalties paid to a creator for a certain NFT. This could be useful in the context of categorising NFTs in terms of royalties. The term “debt“ is used because the standard aims to provide a way of knowing if there are any royalties left unpaid for the NFTs trades that took place in a marketplace that does not support them and, in that case, expose a way of paying them.
+
+Not only the owner of it, but anyone could pay royalties for a certain NFT. This could be a way of supporting a creator for his work.
+
+## Specification
+
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Every contract compliant with [ERC-6786](./eip-6786.md) MUST implement the interface defined as follows:
+
+### Contract Interface
+
+```solidity
+// @title Royalty Debt Registry
+/// Note: the ERC-165 identifier for this interface is 0x253b27b0
+
+interface IERC6786 {
+
+    // Logged when royalties were paid for a NFT
+    /// @notice Emitted when royalties are paid for the NFT with address tokenAddress and id tokenId
+    event RoyaltiesPaid(address indexed tokenAddress, uint256 indexed tokenId, uint256 amount);
+
+    /// @notice sends msg.value to the creator of a NFT
+    /// @dev Throws if there are no on-chain informations about the creator
+    /// @param tokenAddress The address of NFT contract
+    /// @param tokenId The NFT id
+    function payRoyalties(address tokenAddress, uint256 tokenId) external payable;
+
+    /// @notice Get the amount of royalties which was paid for a NFT
+    /// @dev 
+    /// @param tokenAddress The address of NFT contract
+    /// @param tokenId The NFT id
+    /// @return The amount of royalties paid for the NFT
+    function getPaidRoyalties(address tokenAddress, uint256 tokenId) external view returns (uint256);
+}
+```
+
+All functions defined as view MAY be implemented as pure or view
+
+Function payRoyalties  MAY be implemented as public or external
+
+The event RoyaltiesPaid MUST be emitted when the payRoyalties function is called
+
+The supportsInterface method MUST return true when called with `0x253b27b0`
+
+## Rationale
+
+With a lot of places made for trading NFTs dropping down the royalty payment or having a centralised approach, we want to provide a way for anyone to pay royalties to the creators.
```

Comment:
This paragraph belongs in the motivation section because it is justifying the EIP as a whole, and not describing a specific technical choice made within the EIP.